selfservice/strategy/link/strategy_verification.go

```diff
@@ -50,6 +50,8 @@ type verificationSubmitPayload struct {
 	CSRFToken string `json:"csrf_token" form:"csrf_token"`
 	Flow      string `json:"flow" form:"flow"`
 	Email     string `json:"email" form:"email"`
+	// A branding to be applied to the email body
+	Branding string `json:"branding" form:"branding"`
```

Comment:
We have the concept of [`transient_payload`s](https://github.com/ory/kratos/blob/master/selfservice/strategy/password/registration.go#L53) in the registration flow. It's essentially a JSON container for arbitrary data, you can pass through the flow and use in Webhooks. We have it in the backlog to extend that concept to the other flows as well. 

That might be a good fit here too, instead of using a relatively niche parameter on the top level. The piping through to the email templates should be similar to your current approach.

Reply:
Yes, that would work. I think we don't have any special handling of the `transient_payload` parameter (it's literally just passed through), so should almost just be a rename from `Branding` to`TransientPayload`, and it should be good to go.